tests/cassettes/test_tracker_create_list.yaml

```diff
@@ -1,8 +1,8 @@
 interactions:
 - request:
-    body: '{"trackers": {"0": {"tracking_code": "EZ1000000001"}, "1": {"tracking_code":
-      "EZ2000000002"}, "2": {"tracking_code": "EZ3000000003"}, "3": {"tracking_code":
-      "EZ4000000004"}}}'
+    body: !!python/unicode '{"trackers": {"params": {"1": {"tracking_code": "EZ2000000002"},
```

Comment:
Why did this change to unicode?

Reply:
Not sure yet. All I did is re-record the cassette.